packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -646,7 +646,7 @@ export class StripeWebhookHandler extends StripeHandler {
         error,
         plan,
       });
-      reportValidationError(msg, error as any);
+      reportValidationError(Sentry, msg, error as any);
```

Comment:
Sentry is a global, is there any point in passing it around as it'll be the same everywhere when imported?

Reply:
I left a comment in fxa-shared/sentry/index.ts Sentry about this. Basically this allows the Sentry instance to be context agnostic. 

This change was a direct result of running into two issues. First, if I import '@sentry/node' in fxa-shared it ended up causing problems on the client side. This is because fxa-shared/sentry is referenced from a client side context in some places. When @sentry/node is require from a browser context it results in a runtime error. Second, if ignored the first issue and just referenced the global @sentry/node instance in fxa-shared, for some reason the mocks on Sentry in our tests all started failing. 

If this is a no go let me know, and I'd be happy to revisit it, as I am not completely happy with the solution. The browser context issue is easy enough to work around, but the way the mock stopped working was definitely puzzling and might turn into a time suck.

Reply:
Ugh, ok, yea, that's annoying. Thanks for the context on that.